presto-spi/src/main/java/com/facebook/presto/spi/statistics/StatisticsValue.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.statistics;
+
+public final class StatisticsValue
```

Comment:
If this holds just `value` inside, then does it make sense to have such class at all? Having this class makes computations more verbose since you always have to unwrap `value` before doing something with it, e.g:
`a + b = c`
vs
`new StatisticsValue(a.getValue() + b.getValue())`. I think plain double is just nicer to work with.

If we want to add some estimate "quality" metric, then we could create a class: `ConfidenceInterval`, which would provide such "quality". For instance, you could have two fields:
```
double totalRowsEstimate;
ConfidenceInterval totalRowsError;
```
or even simpler we could just always assume confidence 95%, then it would just be:
```
double totalRowsEstimate;
double totalRowsError;
```